language/move-analyzer/editors/code/tests/lsp.test.ts

```diff
@@ -49,4 +57,43 @@ Mocha.suite('LSP', () => {
         assert.deepStrictEqual(syms[0]?.children[3].name, 'this_is_a_test');
         assert.deepStrictEqual(syms[0]?.children[3]?.detail, '["test", "expected_failure"]');
     });
+
+    Mocha.test('textDocument/completion', async () => {
```

Comment:
Could you please include a test for primitive type auto-completion?

Reply:
done!